code/game/objects/items/crayons.dm

```diff
@@ -266,8 +265,12 @@
 	return FALSE
 
 /obj/item/toy/crayon/proc/crayon_text_strip(text)
-	var/static/regex/crayon_r = new /regex(@"[^\w!?,.=%#&+\/\-]")
-	return replacetext(lowertext(text), crayon_r, "")
+	// these replacetexts are removing the < and > symbols after stripped_input()
+	text = replacetext(text, "&gt", "")
+	text = replacetext(text, "&lt", "")
+	text = replacetext(text, "&amp", "&") // we want to re-insert the & symbol
+	var/static/regex/crayon_regex = new /regex(@"[^\w\s!?,.=%#&+/-]", "ig")
+	return lowertext(crayon_regex.Replace(text, ""))
```

Comment:
These new replacements shouldn't be necessary.
If you want `<` and `>` to be deleted instead of changed to the html versions then run the regex before `html_encode()` happens (that means making use of a custom input proc, instead of `stripped_input()`.

Any reason why we'd want to keep `&`, in any case?

The comment on the issue suggests that removal of all spaces is what's intended, so the `"g"` flag for the regex should be enough, the addition of `\s` to the character class changing intended behavior.

On the last `-` in the character class, I know this is not your code, but it'd be better if it was escaped, even if it will work by being put last.